### ISSUE TITLE
Update of the learning parameters after the restore of a checkpoint.

### ISSUE THREAD

**Issue by AGPX** [hill-a/stable-baselines#677]
Hi,

I have to train a network (via PPO2) in different sessions, so I use the checkpoint to load and restore the model. However, in subsequent sessions, it may be necessary to update some learning parameters (such as learning_rate), but unfortunately they are saved with the model. Is there a way to update them after loading the model (simply assigning them to the model doesn't seem to work)?

**Comment by Miffyli** [comment]
Try using `load_parameter` function ([example in docs](https://stable-baselines.readthedocs.io/en/master/guide/examples.html#accessing-and-modifying-model-parameters)). I.e. Create a new agent with new learning rate, and then use `load_parameters` to load the parameters from previous session.
